### ISSUE TITLE
Where to discuss? Also next steps...

### ISSUE THREAD

**Issue by dukedougal** [mattupstate/overholt#30]
Matt I love OverHolt and have built a major application with it.  But I want to know more and discuss it in more detail.

Is there a "next step"? i.e one of your more sophisticated/larger/more advanced projects?

Also is there a better place to raise these sorts of things than as an issue here?

**Comment by mattupstate** [comment]
@dukedougal not really. Overholt is just an illustration of how I developed Flask apps in the past. That said, I may write a new blog post regarding a different approach.

**Comment by dukedougal** [comment]
Hi Matt
My goal was to build better, more organised Flask apps.  The trouble with Flask as that as soon as you get going with it, you want to start doing things in a more organised manner. Overholt is the best organised application for Flask I have yet found.  I have learned a huge amount from following your approach.  I'd be fascinated to hear if you have thoughts on better/alternative ways of doing things.
I really think there should be an official optional framework for building Flask apps and it should be Overholt or something like it. Without Overholt people are just reinventing the wheel.
Is there some way I can get to see the code of a larger application you have built using these techniques? 
thanks
